recipes/pymc3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pymc3" %}
+{% set version = "3.0rc2" %}
+{% set md5 = "80105941758b9eba47c8a2774d5ee905" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11.0
+    - scipy >=0.12.0
+    - matplotlib >=1.5.0
+    - theano >=0.8.2
+    - pandas >=0.18.0
+    - patsy >=0.4.0
+    - joblib >=0.9
+    - tqdm >=4.8.4
+    - sphinx
+    - numpydoc
+
+  run:
+    - python
+    - numpy >=1.11.0
+    - scipy >=0.12.0
+    - matplotlib >=1.5.0
+    - theano >=0.8.2
+    - pandas >=0.18.0
+    - patsy >=0.4.0
+    - joblib >=0.9
+    - tqdm >=4.8.4
+    - sphinx
+    - numpydoc
```

Comment:
I believe you will need to add `enum43  # [py2k]` here.

Also, `sphinx` and `numpydoc` should be used to build the docs, right? Does `setup.py` build the docs by default? If not you probably do not nee them.